src-js/utils.js

```diff
@@ -1,3 +1,10 @@
 export function isProduction() {
   return window.location.origin === 'https://circleci.com';
 }
+
+export function isUnsupportedBrowser() {
+  if (typeof window === 'undefined') return true;
+  const ua = window?.navigator?.userAgent;
+  const msie = ua.indexOf('MSIE ');
+  return msie > 0;
```

Comment:
what foes `msie` stand for in this instance?

Reply:
**M**icro**S**oft **I**nternet **E**xplorer 😃